src/database/DataTypes/ItemStoreJoin.js

```diff
@@ -9,6 +9,7 @@ export class ItemStoreJoin extends Realm.Object {
     // Make the item no longer visible in this store
     const item = itemResults[0];
     item.isVisible = false;
+    item.defaultPrice = 0;
```

Comment:
Not sure why this is necessary? Does mSupply desktop do this? Given visibility is controlled server side, I'd have thought the server would send any update to default price anyway?

Reply:
This is not necessary, defaultPrice would be set to 0 by Item schema declaration. This code is also doing default value set.  
I did this (I think) for two reasons:
1. I was replicating the isVisible code in line 11.
2. It does not hurt the code and acts as a fail safe

Reply:
Yea I guess if the join is recreated then it'll set the value again. mSupply doesn't actually usually delete these joins anyway, on desktop isVisible is on the join and just switches it True or False.